### ISSUE TITLE
Flyway does not execute SQL in a migration to create a MySQL procedure when it contains `DROP TABLE IF EXISTS`

### ISSUE THREAD

**Issue by jamesball-inexus** [flyway/flyway#2706]
###### Which version and edition of Flyway are you using?

`6.2.4`

###### If this is not the latest version, can you reproduce the issue with the latest one as well?

n/a

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)

Gradle plugin

###### Which database are you using (type & version)?

MySQL `8.0.19`

###### Which operating system are you using?

Windows 10

###### What did you do?

Flyway does not execute the SQL in a migration to create a procedure that drops a table if `IF EXISTS` is included in the `DROP TABLE` statement.

```
DELIMITER //
CREATE PROCEDURE drop_table_test ()
SQL SECURITY INVOKER
drop_table_test: BEGIN
    DROP TABLE IF EXISTS test;
END drop_table_test //
DELIMITER ;
```

###### What did you expect to see?

The procedure `drop_table_test` is created.

###### What did you see instead?

The procedure `drop_table_test` is not created.

**Comment by Lyeeedar** [comment]
This will be fixed in 6.3, releasing tomorrow.

Duplicate of https://github.com/flyway/flyway/issues/2687

**Comment by jamesball-inexus** [comment]
@Lyeeedar I have tested this in release `6.3.0` and it is not fixed. The same issue still occurs exactly as before.
